packages/core/src/amazonqFeatureDev/session/sessionState.ts

```diff
@@ -288,6 +300,15 @@ abstract class CodeGenBase {
                 }
                 case CodeGenerationStatus.PREDICT_READY:
                 case CodeGenerationStatus.IN_PROGRESS: {
+                    if (codegenResult.codeGenerationStatusDetail) {
+                        messenger.sendAnswer({
+                            message:
+                                i18n('AWS.amazonq.featureDev.pillText.generatingCode') +
```

Comment:
We previously spoke about potentially splitting the "Generating code..." text into a separate message from the status detail: https://quip-amazon.com/LMbkAG0e56gg#NEJ9DAjvxR0

Did we find that misleading or difficult to produce?

Reply:
Splitting will keep the previous message "Generating code:" in the chat history which not an intended user behaviour. I also don't see a clear benefit for it. Do we have a supporting message to introduce that?